rosie/src/main/java/com/karumi/rosie/domain/usecase/UseCaseHandler.java

```diff
@@ -45,8 +53,12 @@ void execute(RosieUseCase useCase, UseCaseParams useCaseParams) {
     UseCaseFilter.filter(useCase, useCaseParams);
 
     useCase.setOnSuccessCallback(useCaseParams.getOnSuccessCallback());
-    useCase.setOnErrorCallback(
-        new OnErrorCallbackToErrorHandlerAdapter(errorHandler, useCaseParams.getOnErrorCallback()));
+
+    OnErrorCallbackToErrorHandlerAdapter errorAdapter =
```

Comment:
you can use here OnErrorCallback instead OnErrorCallbackToErrorHandlerAdapter and the same with errorAdapters no??